### PR TITLE
feat(Header): Add background image

### PR DESCRIPTION
**Pull request by MHase**
Issue: #1551 
Added possibility to pass `backgroundImage` prop which is simply `source` prop like in `Image` component :) e.g. :
```
source={{ uri: 'http://google.com' }}
```
img_4187

### CODE REVIEW

**Comment by iRoachie** [comment]
Brooooo this PR is 🔥

File: src/header/Header.js [hunk 1]
```diff
@@ -110,6 +112,7 @@ Header.propTypes = {
   rightContainerStyle: ViewPropTypes.style,
   backgroundColor: PropTypes.string,
   backgroundImage: PropTypes.object,
+  backgroundImageStyle: PropTypes.style,
```
Comment (iRoachie) [inline src/header/Header.js]: There's no PropTypes.style.

Comment (MHase) [reply src/header/Header.js]: my bad, I was too quick. do you think simple `PropTypes.object` should be enough?

Comment (iRoachie) [reply src/header/Header.js]: I think ViewPropTypes.style could work or even `Image.propTypes.style`. 

The latter would be more accurate but I am not sure if it works.

**Pull request merged**